hrm-service/src/contact-job/contact-job-complete.ts

```diff
@@ -87,7 +88,19 @@ export const pollAndProcessCompletedContactJobs = async (jobMaxAttempts: number)
 
           return markedComplete;
         } else {
+
           const { jobId, attemptNumber, attemptPayload } = completedJob;
+
+          // emit an error to pick up in metrics since completed queue is our
+          // DLQ. These may be duplicates of ContactJobProcessorErrors that have
+          // already caused an alarm, but there is a chance of other errors ending up here.
+          console.error(
+            new ContactJobCompleteProcessorError(
```

Comment:
This pattern of creating a new error obj inside of a console is a little strange, but it keeps naming conventions for log filters DRY.